components/config/definition.rst

```diff
@@ -231,12 +264,19 @@ In YAML, the configuration might look like this:
 
     database:
         parameters:
-            param1: { value: param1val }
+            parameter1: { value: 'value1' }
+            parameter2: { value: 'value2' }
+
+In XML, the configuration might look like this:
+
+.. code-block:: xml
+
+    ...
```

Comment:
```xml
<parameter value="parameter1">value1</parameter>
<parameter value="parameter2">value2</parameter>
```

All in all, I think you should use `->useAttributeAsKey('name')` or the like and then use the `name` attribute